receiver/datadogreceiver/translator.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package datadogreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/datadogreceiver"
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"mime"
+	"net/http"
+	"strings"
+
+	datadogpb "github.com/DataDog/datadog-agent/pkg/trace/exportable/pb"
+	"github.com/tinylib/msgp/msgp"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	semconv "go.opentelemetry.io/collector/semconv/v1.6.1"
+)
+
+func toTraces(traces datadogpb.Traces, req *http.Request) ptrace.Traces {
+	dest := ptrace.NewTraces()
+	resSpans := dest.ResourceSpans().AppendEmpty()
+	resSpans.SetSchemaUrl(semconv.SchemaURL)
```

Comment:
Traces may come from multiple services, should we split them into different resources spans?

Reply:
Please, share what's the resolution to this.

Reply:
Not resolving in this PR

Reply:
The reason why we dont this is because we dont set any attrs on the ResourceSpan object, only actually spans